pages/account_and_service_management/account_information/iam-policies-api/guide.en-gb.md

```diff
@@ -159,7 +167,36 @@ For example, create a policy to authorise a user named "*user1*" to do some acti
 
 This policy is not for an account but for a user, so the identity URN corresponds to the format "*urn:v1:eu:identity:**user**:**account-id**/**username***".
 
-With this JSON as body for the `POST /iam/policy` call, the policy will be created.
+```json
+{
+    "description": "VPS - all except delete snapshot",
+    "identities": [
+        "urn:v1:eu:identity:user:xx1111-ovh/user2"
+    ],
+    "name": "vps-all-but-delete-snapshot",
+    "permissions": {
+        "allow": [
+            {
+                "action": "vps:apiovh:*"
+            }
+        ],
+        "except": [
+            {
+                "action": "vps:apiovh:snapshot/delete"
+            }
+        ]
+    },
+    "resources": [
+        {
+            "urn": "urn:v1:eu:resource:vps:vps-5b48d78b.vps.ovh.net"
+        }
+    ]
+}
+```
+
+This policy uses a wildcard with an except. This allows to easily give a large set of permissions. Here the user `user2` will have all permissions of the **vps** type on `vps-5b48d78b.vps.ovh.net` except for the permission to delete a snapshot.
```

Comment:
What about using this example to demonstrate wildcard in both actions & resources ?

```suggestion
This policy uses a wildcard  on allowed actions and exclude a few with an except statement. This helps giving a large set of permissions quickly. In this example user `xx1111-ovh/user2` will have every permissions of the **vps** type on  every  VPS of xx1111-ovh, except for the permission to delete a snapshot.
```